src/content/docs/examples/releases-content-layer.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: List recent releases with the Content Layer API
+description: A custom page that lists recent releases for a GitHub repository using the Content Layer API
+---
+
+import StackBlitz from "../../../components/StackBlitz.astro";
+
+This example demonstrates a custom page, which uses the experimental [Content Layer API](https://astro.build/blog/future-of-astro-content-layer/) to list recent releases for a GitHub repository.
+
```

Comment:
I think we should definitely include a link to your blog post. Maybe something like this?

```suggestion

Read [a full write-up of this technique](https://hideoo.dev/notes/starlight-list-recent-github-releases) on HiDeoo’s blog.

```